pkg/operator/ceph/object/rgw.go

```diff
@@ -117,11 +113,15 @@ func (c *clusterConfig) startRGWPods() error {
 			DaemonID:     daemonName,
 		}
 
-		// Generate the keyring after starting the replication controller so that the keyring may use
-		// the controller as its owner reference; the keyring is deleted with the controller
-		keyring, err := c.generateKeyring(rgwConfig)
+		// We now set the owner reference to the CRD controller instead of the replicaset, this both pros and cons
```

Comment:
I'd vote to simplify the comments and no need to save the pros and cons of doing it the old vs the new way. We can just explain the new reasons.

Reply:
Ok will shrink a bit.